### PR TITLE
Create a checklist

### PR DESCRIPTION
**Pull request by LouiseBonnard**
Create a checklist to make sure specifications are as complete as possible. I may have forgotten some parts, feel free to add it to this file if you think of something else. ;-)

### CODE REVIEW

File: CHECKLIST.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Finally, **the key element is the user story**. Feel free to add it whenever it is necessary following this template: _As a [user], I want to [be able to do sthing] so that [I can achieve my outcome]._
+
+
+## Functional behaviors
+
+- [ ] **Mockups**, if there are any links to share.
+
+- [ ] **Helper cards**, if there are any.
+
+- [ ] **Empty spaces**, if there are any.
```
Comment (marionf) [inline CHECKLIST.md:28]:
```suggestion
- [ ] **Empty states**, appears only when there is no value, they should be specified if there are any.
```
I guess you mean empty states ?

File: CHECKLIST.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- [ ] **Descriptions**, note how to use it: invalid characters, maximum length, actions triggered, etc.
+
+- [ ] **Multilanguages**, if this component can be localized according to the store's languages.
+
+- [ ] **Help notes**, write down the expected available tooltips, placeholders, help texts, etc.
+
+- [ ] **Notifications**, add the warning, success, error, and information notifications displayed.
+
+- [ ] **Issues**, link the related bugs or improvements it specified if there are any.
+
```
Comment (marionf) [inline CHECKLIST.md:43]:
```suggestion
- [ ] **RTL**, if the page has some specifics behaviors in a right-to-left language

- [ ] **Search & sort**, tables columns often have search fields and sort, this should be specified for each column if there are.

- [ ] **Bulk actions**, table columns often have bulk-actions drop-down, this should be specified if there are.

- [ ] **Settings wheel**, tables often have a settings wheel with options, this should be specified if there are.
```

File: CHECKLIST.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+If the specifications are a bit long or describe several different cases, **do not hesitate to add a table of contents with anchored links** or explain how you structured the file.
+
+Finally, **the key element is the user story**. Feel free to add it whenever it is necessary following this template: _As a [user], I want to [be able to do sthing] so that [I can achieve my outcome]._
+
+
+## Functional behaviors
+
+- [ ] **Mockups**, if there are any links to share.
+
+- [ ] **Helper cards**, if there are any.
```
Comment (marionf) [inline CHECKLIST.md:26]:
```suggestion
- [ ] **Helper cards**, help merchants to have a better understanding of back office pages, they should be specified if there are any.
```
Maybe we can detail what is an helper card ?

**Review by marionf — COMMENTED** [review]
I added some suggestions @LouiseBonnard ;)

File: CHECKLIST.md (hunk 1 above)
Comment (LouiseBonnard) [reply CHECKLIST.md:28]: 🙄 Of course, thanks a lot @marionf!

**Review by marionf — APPROVED** [review]
Thank you @LouiseBonnard :tada:

**Comment by marionf** [comment]
Thank you @LouiseBonnard :)

**Pull request merged**